systemverilog-plugin/UhdmAst.cc

```diff
@@ -3587,8 +3587,15 @@ void UhdmAst::process_assignment_pattern_op()
         visit_one_to_many({vpiOperand}, obj_h, [&](AST::AstNode *node) {
             if (node->type == AST::AST_ASSIGN || node->type == AST::AST_ASSIGN_EQ || node->type == AST::AST_ASSIGN_LE) {
                 // Find at what position in the concat should we place this node
```

Comment:
Could you please modify this comment or add another one here? Honestly I didn't really help me understand what happens here, and I think it would be beneficial for us to have an explanation of what happens for each of the `vpiOperand`.

Reply:
I added some comments, PTAL to let me know if it makes the operations more understandable.

Reply:
Thanks a lot!